.build_scripts/generateTasks.js

```diff
@@ -60,8 +60,9 @@ var fetchProjectData = function(projectNumber, cb) {
 
       if(jsonResponse){
         /// capitalization or presence/lack of a space in Missing Maps shouldn't matter
-        var nameCheck = jsonResponse.name.replace(/\s+/g, '').toLowerCase().indexOf("missingmaps");
-        if(nameCheck !== -1 && jsonResponse.status == 'PUBLISHED'){
+        var nameCheck = jsonResponse.name.replace(/\s+/g, '').toLowerCase().indexOf("");
+        // if(nameCheck !== -1 && jsonResponse.status == 'PUBLISHED'){
+        if(jsonResponse.status == 'PUBLISHED'){
```

Comment:
What are these changes for?

Reply:
the code is based off stuff in the below? (based on what it was before i recently updated it)
https://github.com/AmericanRedCross/osm-data-parse/blob/master/tasking-mgr-projects/parse.js

not sure how it's being used here. and why the above edit removes the check for "missingmaps" in the title.

missing maps tasks don't start until 718.

i think this code will move to `makeTasks` before all the `fetchProjectData` inside `throttleProjects` return.

Reply:
Desire to remove the missing maps tasks was because other people use the TM. The microsites are for more than just missing maps stuff. We do need a more elegant way of doing this but this is what we got for the moment. I'm open to suggestions. 

Long term these pages will need links to a new TM for normal mapping purposes such as country wide road and settlement tasks.